.github/workflows/integration-test.yml

```diff
@@ -44,7 +44,11 @@ jobs:
           cat $PORTAL_SOURCE_DIR/src/main/resources/application.properties | \
               sed 's|spring.datasource.mysql.url=.*|spring.datasource.mysql.url=jdbc:mysql://cbioportal-database:3306/cbioportal?useSSL=false|' | \
               sed 's|spring.datasource.mysql.username=.*|spring.datasource.mysql.username=cbio_user|' | \
-              sed 's|spring.datasource.mysql.password=.*|spring.datasource.mysql.password=somepassword|' \
+              sed 's|spring.datasource.mysql.password=.*|spring.datasource.mysql.password=somepassword|' | \
+              sed 'a spring.datasource.driver-class-name=com.mysql.jdbc.Driver' | \
+              sed 'a spring.datasource.url=jdbc:mysql://cbioportal-database:3306/cbioportal?useSSL=false' | \
+              sed 'a spring.datasource.username=cbio_user' | \
+              sed 'a spring.datasource.password=somepassword' \
```

Comment:
This is a workaround to add legacy properties which are needed by `migrate.py` script. This is required for now because migration script is in the separate cbioportal-core repo.